src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -683,10 +754,13 @@ public async Task VerifyBankAsync(Guid organizationId, int amount1, int amount2)
         await _referenceEventService.RaiseEventAsync(
             new ReferenceEvent(ReferenceEventType.Signup, organization, _currentContext)
             {
-                PlanName = plan.Name,
-                PlanType = plan.Type,
+                PlanName = passwordManagerPlan.Name,
+                PlanType = passwordManagerPlan.Type,
                 Seats = returnValue.Item1.Seats,
                 Storage = returnValue.Item1.MaxStorageGb,
+                SmSeats = returnValue.Item1.SmSeats,
+                ServiceAccounts = returnValue.Item1.SmServiceAccounts
+
```

Comment:
We might need to add "UseSecretsManager = signup.UseSecretsManager"

Reply:
Added